packages/infra/lib/api-stack.ts

```diff
@@ -767,6 +779,58 @@ export class APIStack extends Stack {
     fitbitResource.addMethod("GET", new apig.LambdaIntegration(fitbitSubscriberVerificationLambda));
   }
 
+  private setupTenoviWebhook(ownProps: {
+    lambdaLayers: lambda.ILayerVersion[];
+    baseResource: apig.Resource;
+    secrets: Secrets;
+    vpc: ec2.IVpc;
+    fargateService: ecs_patterns.NetworkLoadBalancedFargateService;
+    tenoviAuthHeader: string;
+    envType: string;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }) {
+    const {
+      lambdaLayers,
+      baseResource,
+      secrets,
+      vpc,
+      fargateService: server,
+      tenoviAuthHeader,
+      envType,
+      sentryDsn,
+      alarmAction,
+    } = ownProps;
+
+    const tenoviAuthLambda = createLambda({
+      stack: this,
+      name: "TenoviAuth",
+      runtime: lambda.Runtime.NODEJS_18_X,
+      entry: "tenovi",
+      layers: lambdaLayers,
+      envVars: {
+        API_URL: `http://${server.loadBalancer.loadBalancerDnsName}/webhook/tenovi`,
+        ENV_TYPE: envType,
+        TENOVI_AUTH_HEADER: tenoviAuthHeader,
+        ...(sentryDsn ? { SENTRY_DSN: sentryDsn } : {}),
+      },
+      vpc,
+      alarmSnsAction: alarmAction,
+    });
+
+    type SecretKeys = keyof EnvConfig["providerSecretNames"];
```

Comment:
Why this? I see its being used on `setupFitbitWebhook`, but it doesn't seem necessary - I just removed those local and it worked fine.

If we really don't need, let's remove it here and there while we're at it, please?